index.js

```diff
@@ -14,9 +14,17 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell
-//. and PureScript. It's stricter and more opinionated than [Ramda][],
-//. and provides a similar suite of functions.
+//. Sanctuary is a JavaScript functional programming library inspired
+//. by Haskell and PureScript. It's stricter and more opinionated than
```

Comment:
It might mean that Sanctuary has a bigger impact on how one structures their code, in which case it may be more informative to add that to the introduction instead?